src/demo/demo.ejs

```diff
@@ -12,7 +12,7 @@
       default-src 'self' https://assets.onfido.com;
       script-src 'self' https://www.woopra.com https://assets.onfido.com https://sentry.io;
       style-src 'self' 'unsafe-inline' https://assets.onfido.com;
-      connect-src blob: *.onfido.com wss://*.onfido.com *.dev.onfido.xyz wss://*.dev.onfido.xyz https://www.woopra.com https://sentry.io;
+      connect-src blob: <%= process.env.NODE_ENV === 'test' ? 'https://localhost:8080' : '' %> *.onfido.com wss://*.onfido.com *.dev.onfido.xyz wss://*.dev.onfido.xyz https://www.woopra.com https://sentry.io;
```

Comment:
:+1: 

i wouldn't oppose on simply adding localhost:8080.

This is only for testing purposes so it would be ok